src/pr-info.ts

```diff
@@ -159,6 +162,12 @@ export interface PrInfo {
     readonly reviews: readonly ReviewInfo[];
 }
 
+export type NotFail =
```

Comment:
`BotNotFail` is much better.

Maybe also a `BotResult` instead of `NotFail | BotFail`?

Reply:
Done :heavy_check_mark: